README.md

```diff
@@ -789,6 +790,12 @@ Supported `temp` values: 1-65279 mired. Actual color temperature supported by en
 
 - Mix of warm white intensity and cold white intensity forms color temperature.
 
+- Duration property within cmd.color.start_transition command is optional and means seconds.
+
+- For Z-Wave protocol, every duration value above the 'maxSeconds' value will not be handled and will be cut into full minutes.
+  Example: sending "duration": 128 (seconds = 2 minutes and 8 seconds) will result in adapter setting only 2 minutes within device.
+  Duration increment step should be '1' for seconds: 1, 2, 3...125, 126, 127 and '60' for minutes, starting from: 180, 240, 300...7620.
```

Comment:
This seems extreme. I would prefer a solution that's a bit more straight forward.

Reply:
More like steps:
sup_durations: [
{min: 1, max: 127, step: 1},
{min: 180, max: 7620, step: 60}
]

Reply:
Alternatively we can just have 

`sup_durations: {min: 1000, max: 999999}`

Where the duration would always be in milliseconds. I don't think the protocol should worry about how things are to be displayed. Is there a technical reason why it's split into seconds and minutes?

Reply:
@boguslaw-wojcik seems nice, thank You, will fix it

Reply:
fixed